scripts/upgrade_samples.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# Copyright (C) 2020 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+########################################################################
+#
+# Changes all samples to a new Compose version
+#
+# Example: To run 
+#     ./scripts/upgrade_samples.sh
+#
+########################################################################
+
+set -e
+
+echo "Version to change Compose to (e.g 1.2.0-alpha06): ";
+read compose_version;
+
+echo "Snapshot ID: (Blank for none)";
+read snapshot_version;
+
+echo "Upgrading to $compose_version snapshot $snapshot_version"
```

Comment:
Nit: Maybe just a quick if statement here to print the snapshot portion only if there is a snapshot specified (so we don't have output like "Upgrading to 1.2.3 snapshot " when snapshot is null/empty).

Reply:
Done